docs/site/Serving-static-files.md

```diff
@@ -68,3 +80,24 @@ await app.boot();
 await app.start();
 app.static('/files', path.join(__dirname, 'files'));
 ```
+
+### Authentication
+
+The static router
+[does not](https://github.com/strongloop/loopback-next/issues/1144#issuecomment-438359985)
+work smoothly with authentication. If you are using authentication, then you
+should change the way `authenticateRequest` is called in `sequence.ts` to
+operate like this:
+
+```ts
+  async handle(context: RequestContext) {
+    // ...
+
+      // Only attempt authentication on non-static routes, because authenticateRequest fails on static routes
+      if (!(route instanceof StaticAssetsRoute)) {
```

Comment:
Can we instead fix `@loopback/authentication` please? I understand this doc change is a quick improvement, but I think everybody would be better of if we fixed the problem properly.

Reply:
Re-posting a part of https://github.com/strongloop/loopback-next/issues/1144#issuecomment-438368225

Most importantly, I think we need to fix our reference `authenticateRequest` function to correctly support `StaticAssetsRoute` and any other custom route that apps or extensions my register via `app.route()` API.